src/data/automation.ts

```diff
@@ -172,6 +172,46 @@ export type Condition =
   | DeviceCondition
   | LogicalCondition;
 
+export interface EventAction {
```

Comment:
These are shared between automations and scripts, so maybe better fit in `data/script` ? (as we call it the script syntax internally)